### PR TITLE
fix mint fee decimal

### PR DESCRIPTION
**Pull request by 0xaptosj**
we store APT in smallest unit (10e-8) onchain

### CODE REVIEW

File: templates/digital-asset-template/frontend/pages/CreateCollection.tsx [hunk 1]
```diff
@@ -250,7 +251,8 @@ export function CreateCollection() {
                       className={cn(
                         "w-[280px] justify-start text-left font-normal",
                         !publicMintEndDate && "text-muted-foreground"
-                      )}>
+                      )}
+                    >
```
Comment (0xaptosj) [inline templates/digital-asset-template/frontend/pages/CreateCollection.tsx:255]: this is my auto formatter

**Review by 0xmaayan — APPROVED** [review]

File: templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx [hunk 2]
```diff
@@ -145,7 +153,7 @@ export function CreateFungibleAsset() {
               <Input
                 type="number"
                 onChange={(e) => {
-                  setCostToMint(e.target.value);
+                  setMintFeePerFA(parseInt(e.target.value));
```
Comment (0xmaayan) [inline templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx:156]: why this change? label says "cost to mint" but event sets mint fee per fa.

we should keep the same naming

File: templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx [hunk 3]
```diff
@@ -46,15 +45,24 @@ export function CreateFungibleAsset() {
...
       maxMintPerAccount
     ) {
       setDisableCreateAssetButton(false);
     }
-  }, [name, symbol, maxSupply, costToMint, decimal, iconURL, projectURL]);
+  }, [
+    name,
+    symbol,
+    maxSupply,
+    mintFeePerFA,
```
Comment (0xmaayan) [inline templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx:60]: is "mint fee per fa" / "cost to mint" a required field? can i create a FA that is free to mint?

File: templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx (hunk 2 above)
Comment (0xaptosj) [reply templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx:156]: nft template UI says `MintFeePerNFT`

File: templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx (hunk 3 above)
Comment (0xaptosj) [reply templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx:60]: you can set it to 0 to make it free

File: templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx (hunk 2 above)
Comment (0xmaayan) [reply templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx:156]: then lates change the UI label here also? https://github.com/aptos-labs/create-aptos-dapp/pull/68/files#diff-01771c38a69a7a89669fad355a6d0b769a3de007ed1560828051d77be0bd023eR152

Comment (0xaptosj) [reply templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx:156]: done

File: templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx (hunk 3 above)
Comment (0xmaayan) [reply templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx:60]: i see. then it probably shouldn't be required and the contract can set it if it is undefined. not a blocker tho
